crates/ruff/src/message/text.rs

```diff
@@ -114,8 +114,8 @@ impl Emitter for TextEmitter {
 }
 
 pub(super) struct RuleCodeAndBody<'a> {
-    pub message_kind: &'a DiagnosticKind,
-    pub show_fix_status: bool,
+    pub(crate) message_kind: &'a DiagnosticKind,
```

Comment:
Can it be avoided that it adds the pub(crate) if the type itself isn't visible?

Reply:
Currently, it seems not.
I could not find an issue concerning this in rust-lang/rust.